test/objectFifo-stateful-transform/AIE2_cyclostatic_l2.mlir

```diff
@@ -0,0 +1,95 @@
+// In this example, an AIE core pushes data into a memtile, in a one-by-one
+// fashion. The memtile forwards this one-by-one to a consumer tile. The 
+// consumer tile cyclostatically consumes {1, 2, 1} elements at a time.
+
+// RUN: aie-opt --aie-objectFifo-stateful-transform %s | FileCheck %s
+
+// This test does not currently pass, because objectFifo.release cannot be
+// used in conjunction with objectFifo.link.
+
+module @aie2_cyclostatic_l2 {
+    AIE.device(xcve2302) {
+
+        %tile22 = AIE.tile(2, 2)  // producer tile
+        %memtile = AIE.tile(1, 2) // mem tile
```

Comment:
I think you may have your indices flipped. MemTile would be `AIE.tile(2,1)`

Reply:
Indeed, this test fails with this combination because the lowering identifies shared memory between %tile22 and %memtile. If you switch it around, then it behaves correctly with respect to the objectFifo lowering and what it currently supports.

Reply:
fixed! thanks for noticing this